manifests/kuadrant/istio/istio.yaml

```diff
@@ -8,11 +8,31 @@ metadata:
 spec:
   version: v1.23.0
   namespace: istio-system
-  # Disable autoscaling to reduce dev resources
   values:
+    global:
+      defaultPodDisruptionBudget:
+        enabled: true
     pilot:
-      autoscaleEnabled: false
-
+      autoscaleMin: 2
+      autoscaleMax: 4
+      # [NOTE] values set to minimize resource consumption on a local env
+      resources:
+        requests:
+          cpu: 10m
+          memory: 10Mi
+      topologySpreadConstraints:
+        - maxSkew: 1
+          topologyKey: kubernetes.io/hostname
+          whenUnsatisfiable: ScheduleAnyway
+          labelSelector:
+            matchLabels:
+              app: istiod
+        - maxSkew: 1
+          topologyKey: kubernetes.io/zone
+          whenUnsatisfiable: ScheduleAnyway
+          labelSelector:
+            matchLabels:
+              app: istiod
```

Comment:
Are these meant as 2 separate examples, or is the combination of both a desired setup?
In the first constraint, istiod will (ideally) be deployed on each node.
The second constraint is lesser, in that it could lead to less istiod pods running than the number of nodes (e.g. if you have 6 nodes, but only 3 zones).
Would the second constraint be rudantant in this case?

Reply:
Thinking more on this, it probably depends on the min/max, and which constraint is satisfied first
```
autoscaleMin: 2
autoscaleMax: 4
```